libs/ui-watt/src/lib/foundations/typography/+storybook/storybook-typography-overview.component.html

```diff
@@ -19,20 +19,49 @@ <h1>Typography</h1>
 </mat-card>
 
 <mat-card>
-  <h3>Font</h3>
-  <p>
-    <strong>Open Sans</strong> is the typeface used in the Watt Design System.
-  </p>
+  <h3>Font Family Options</h3>
+  <ul>
+    <li>Primary: <strong>Open Sans</strong></li>
+    <li>Secondary: <strong>Open Sans</strong></li>
+  </ul>
 </mat-card>
 
 <mat-card>
-  <h3>Type Scale</h3>
-  <p>
-    Font sizes are smaller on mobile and scale up at the md breakpoint to be
-    larger on desktop.
-  </p>
+  <h3>Font Weight Options</h3>
+  <ul>
+    <li class="font-weight-light">Light: 300</li>
+    <li class="font-weight-regular">Regular: 400</li>
+    <li class="font-weight-semi-bold">Semi-bold: 600</li>
+    <li class="font-weight-bold">Bold: 700</li>
+  </ul>
+</mat-card>
+
+<mat-card style="display: flex; gap: 64px">
+  <section>
+    <h3>Font Size Options</h3>
+    <p>size (line-height)</p>
```

Comment:
nitpick: The line-height value is also in px. Let's reflect that here. 

suggestion: Use `font-size/line-height` or `font-size (line-height)` format.